openxla-nvgpu/compiler/src/openxla/compiler/nvgpu/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("@iree_core//build_tools/bazel:build_defs.oss.bzl", "iree_compiler_register_plugin")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+cc_library(
+    name = "defs",
+    includes = ["../../.."],
+)
+
+cc_library(
+    name = "registration",
+    srcs = [
+        "PluginRegistration.cpp",
+    ],
+    deps = [
+        ":defs",
+        "//compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/IR",
```

Comment:
My preference is `CuDNN` (file names, dialect/ops prefix, etc...) to be consistent with future `CuSolver`, `CuSparse`, etc... (I can send a patch for it once I'll start adding new ops and passes)

Reply:
Let's followup with a patch. This was a copy-paste from the existing directory and it should be easy to rename once landed.

Reply:
SG (cuDNN dir sounds fine, can be changed, it is TBD how many will just be custom calls vs dialects proper still, it's an easy sed script before graduation:-))